config.make.in

```diff
@@ -60,6 +60,11 @@ VERSION = 2.3.98.1
 TARGET = monotouch
 endif
 
+ifeq (x-$(TargetFramework)-$(FSharpCoreBackVersion),x-xamarinwatch-)
+VERSION = 2.3.98.1
```

Comment:
It sounds safer, but I don't know what kind of problems different version numbers can cause.

FWIW xamarinwatchos is be closer to monotouch than monodroid is, which is why I went with the same version number in the first place.